machine_learning/scoring_functions.py

```diff
@@ -135,3 +135,10 @@ def mbd(predict, actual):
     score = float(numerator) / denumerator * 100
 
     return score
+
+
+def manual_accuracy(predict, actual):
+    predict = np.array(predict)
+    actual = np.array(actual)
+    score = np.mean(actual == predict)
+    return score
```

Comment:
```suggestion
    return np.mean(np.array(actual) == np.array(predict))
```

Reply:
Yep we can do this by just writing a single line.
Thanks @cclauss